### PR TITLE
[Asset] Improved base path normalization

### PR DESCRIPTION
**Pull request by dosten**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| License | MIT |

A minor fix in the normalization of the base path

### CODE REVIEW

**Comment by dosten** [comment]
The failing tests aren't related.

File: src/Symfony/Component/Asset/PathPackage.php [hunk 1]
```diff
@@ -39,11 +39,7 @@ public function __construct($basePath, VersionStrategyInterface $versionStrategy
         if (!$basePath) {
```
Comment (Tobion) [inline src/Symfony/Component/Asset/PathPackage.php]: this looks wrong fro `'0'`

Comment (dosten) [inline src/Symfony/Component/Asset/PathPackage.php]: Yes, you're right. I can open another PR to solve this.

**Comment by dosten** [comment]
I'm closing this because if the base path is "/" a wrong base path is generated.

**Pull request closed without merging**